python/tstUtilities.py

```diff
@@ -1039,6 +1059,12 @@ def processTstLine(enviroPath, line):
             returnData = processSlotLines(api, pieces, triggerCharacter)
         elif line.upper().startswith("TEST.REQUIREMENT_KEY"):
             returnData = processRequirementLines(api, pieces, triggerCharacter)
+        elif line.upper().startswith("TEST.SUBPROGRAM"):
+            if unit == None:
+                globalOutputLog.append(
+                    "Additional 'unit' parameter is requiered for TEST.SUBPROGRAM: autocompletion."
```

Comment:
required

Reply:
Changed in 265346f47f3d1a2bda59ea03cf5c5e455c889ea0